ztp/source-cluster-crs/cluster-crs.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: v1
```

Comment:
Why are we cloning these ?

Reply:
Those are the source that we use to generate every cluster CRs

Reply:
every field in those CRs has equivalent in th SiteConfig CR. the generator use this file to know the mapping between those CRs fields and the SitcConfig fields. If one of those CRs having a new field, we will need to add this new field and update the SiteConfig CR as well